app/lib/notifications.server.ts

```diff
@@ -1,6 +1,7 @@
 import { createTransport } from 'nodemailer';
 import { secrets } from 'docker-secret';
 import logger from './logger.server';
+require('dotenv').config();
```

Comment:
The rest of our code is doing this at the level of the script, see https://github.com/Seneca-CDOT/starchart/blob/main/package.json#L23-L24.  Let's do the same for the `test` script.